src/DetailsView/reports/components/report-sections/header-section.scss

```diff
@@ -29,10 +29,11 @@
         margin: 0px 0px 0px 16px;
         display: inherit;
 
-        color: $neutral-60;
+        color: $primary-text;
```

Comment:
It looks like the color used for the header text in details view is `neutral-100` (inherited from `body` in `detailsview.scss`) but I don't know if that matters.

Reply:
Good catch. I'll have a closer look, make the color on both places the same.

Reply:
checked with @cheade and we want `$primary-text` on both places. Also, we want to use `font-weight: normal;` on both place.